base/applications/regedit/find.c

```diff
@@ -223,14 +223,25 @@ BOOL RegFindRecurse(
                                   NULL, &cb);
         if (lResult != ERROR_SUCCESS)
             goto err;
-        pb = malloc(cb);
+        pb = malloc(cb + 3); /* To avoid buffer overrun, append 3 NULs */
         if (pb == NULL)
             goto err;
         lResult = RegQueryValueExW(hSubKey, ppszNames[i], NULL, &type,
                                   pb, &cb);
         if (lResult != ERROR_SUCCESS)
             goto err;
 
+        /* Test for odd (not even) number of bytes returned */
+        if ((cb / 2) * 2 != cb)
```

Comment:
```txt
if (cb & 1)
```

Reply:
Done. Your suggestion is much simpler. Thanks.